src/util/githubFetcher.ts

```diff
@@ -10,88 +10,89 @@ const PACKAGE_JSON = '/master/package.json';
 const GITHUB = 'github.com';
 const RAW_GITHUB = 'raw.githubusercontent.com';
 
-class GithubFetcher {
-  /**
-   * Gets all repositories from given github organization.
-   * @param organization organization to look at.
-   */
-  async getRepositories(
-    organization: string,
-    options?: Options
-  ): Promise<JSObject[]> {
-    const repositories: JSObject[] = [];
-    const axiosOptions = {
-      headers: {
-        Authorization: options?.token ? `token ${options.token}` : '',
-      },
-    };
-    // collect all repositories from github
-    let page = 1;
-    let pageIsEmpty = false;
-    while (!pageIsEmpty) {
-      const { data, status } = await axios.get(
-        `https://api.github.com/orgs/${organization}/repos?per_page=50&page=${page}`,
-        axiosOptions
-      );
-
-      if (status !== 200 && status !== 404) {
-        throw new Error(
-          data.message || 'An error occurred when fetching from GitHub'
-        );
-      }
+/**
+ * Gets all repositories from given github organization.
+ * @param organization organization to look at.
+ */
+async function getRepositories(
+  organization: string,
+  options?: Options
+): Promise<JSObject[]> {
+  const repositories: JSObject[] = [];
+  const axiosOptions = {
+    headers: {
+      Authorization: options?.token ? `token ${options.token}` : '',
+    },
+  };
+  // collect all repositories from github
+  let page = 1;
+  let pageIsEmpty = false;
+  while (!pageIsEmpty) {
```

Comment:
While this is ok one can use page_count to paginate

Reply:
Can you give me more info on what you mean by using `page_count` ??